src/actions/index.js

```diff
@@ -8,7 +8,7 @@ export const removeBook = (book) => ({
   payload: book,
 });
 
-export const filterChange = (filt) => ({
+export const changeFilter = (filt) => ({
   type: 'Change',
```

Comment:
- [ ] it adds `CHANGE_FILTER` action [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/HandelChangeFilter.png). Please change` type: 'Change',` to `type: 'CHANGE_FILTER',`